docker-compose.override.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  express-server:
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}:${IMAGE_VERSION}
+    depends_on:
+      - postgres-primary-db
+    command: ["npm", "run", "dev"]
+    container_name: ${IMAGE_NAME}
+    volumes:
+      - ".:/src/app:rw"
+    env_file: .env
+    environment:
```

Comment:
Don't need `environment` settings, keep everything in `.env` file, easier to manage environment in one place.